homeassistant/components/airnow/strings.json

```diff
@@ -25,6 +25,13 @@
     "sensor": {
       "o3": {
         "name": "[%key:component::sensor::entity_component::ozone::name%]"
+      },
+      "station": {
+        "name": "PM2.5 reporting station",
+        "state_attributes": {
+          "lat": { "name": "Latitude" },
+          "long": { "name": "Longitude" }
```

Comment:
```suggestion
          "lat": { "name": "[%key:common::config_flow::data::latitude%]" },
          "long": { "name": "[%key:common::config_flow::data::longitude%]" }
```

Reply:
I explicitly didn't do this since this is a sensor attribute and not a config flow data thing, would you agree otherwise?

Reply:
I think on this occasion context can't be different e.g. different translations and I did check the code base that other integrations does this as well.
So I would vote against myself on this particular one.